win32/scalyr-agent.spec

```diff
@@ -20,7 +20,19 @@ from PyInstaller.utils.hooks import collect_submodules
 
 block_cipher = None
 
-windows_monitors = ['windows_process_metrics', 'windows_system_metrics', 'windows_event_log_monitor']
+windows_monitors = [
```

Comment:
This is a duplicate of what we do here: https://github.com/scalyr/scalyr-agent-2/blob/master/setup.py#L78

I suspect there's not an easy way to have both setup.py and this file point at a common definition of this.  If that's true, can you at least add comments in both places reference to the other location so we know we have to update both?

You should know I also created a static pylint checker that made sure the one in setup.py is up to date. So, if we ever get rid of the setup.py, we should change that checker to look at this location.  You could add that to the comment.  See this: https://github.com/scalyr/scalyr-agent-2/blob/master/pylint_plugins/py2exe_checker.py#L44